### ISSUE TITLE
Installation instuction 

### ISSUE THREAD

**Issue by nscpro** [ffplayout/ffplayout#397]
is it possible for someone to make a short video how to install it on ubuntu linux. Using the command line, you cannot install a .rpm or .deb file. The installation instructions are so confusing and incomprehensible that what is written there does not correspond to reality.

**Comment by jb-alvarado** [comment]
Hello @nscpro, way is not possible to install the .deb file? And if you tell us, on which step you hang, I belief we can help you.

**Comment by nscpro** [comment]
> Hello @nscpro, way is not possible to install the .deb file? And if you tell us, on which step you hang, I belief we can help you.
> 
> And have you read this [instruction](https://github.com/ffplayout/ffplayout/blob/master/docs/install.md)? When I fly over it again, I did not see something missing.

I’m using Ubuntu Server 22.04 and if I use “apt install <install_file>.deb in terminal command line, it says error.

**Comment by jb-alvarado** [comment]
Your downloaded deb file is still in root folder, you have to move it to the **/tmp/** folder. Then you can  run the install command again. Is better to install the file from tmp folder, then apt will not complain.

**Comment by nscpro** [comment]
> Your downloaded deb file is still in root folder, you have to move it to the **/tmp/** folder. Then you can run the install command again. Is better to install the file from tmp folder, then apt will not complain.

Okay, thats work but i have so many errors. How to fix them? Maybe somebody help to configuration

![IMG_1237](https://github.com/ffplayout/ffplayout/assets/16063685/54f823e7-e578-4d01-8fe8-d4853f3518ea)

**Comment by jb-alvarado** [comment]
The error are not a problem. They only say, that:

1. there is no playlist, so you have to create on
2. there is no filler clip set, you can set one under storage section, or leaf it blank

**Comment by nscpro** [comment]
Okay, but I don't understand, why mp4 file not working? If I open server or pc, it's working fine
<img width="1142" alt="Screenshot 2023-10-06 at 13 55 30" src="https://github.com/ffplayout/ffplayout/assets/16063685/5edf257a-d3c0-47f6-aefb-332e458e4ceb">

**Comment by jb-alvarado** [comment]
Which Codec uses this mp4 file? The frontend can only previewing files, which are supported by the browser.

**Comment by nscpro** [comment]
> Which Codec uses this mp4 file? The frontend can only previewing files, which are supported by the browser.

I’m using H.264/MPEG-4 codec. If I look preview in Media section, it’s working, only Player section preview not working

**Comment by jb-alvarado** [comment]
Ok, that was a bug which was fixed already in upcoming version 0.20. But the version is still not ready, so I create a bug fix release 0.19.1. It will be online in a half hour.

**Comment by JunioCalu** [comment]
> is it possible for someone to make a short video how to install it on ubuntu linux. Using the command line, you cannot install a .rpm or .deb file. The installation instructions are so confusing and incomprehensible that what is written there does not correspond to reality.

Based on my experience with the software, I follow the following installation steps:

```
sudo apt install ./ffplayout_x.xx.x.deb

#Enabling the ffplayout API as a system startup service
sudo systemctl enable --now ffpapi.service

#To create the database
sudo ffpapi -i

#To create an administrative user
sudo ffpapi -a

#Enabling the first channel's web frontend as a system startup service
sudo systemctl enable --now ffplayout.service

#Optional: To start the second channel
#sudo systemctl enable --now ffplayout@channel002.service

#To fix the .ffp_status file creation error
sudo mkdir -v /home/ffpu/
sudo chown -Rv ffpu:nogroup /home/ffpu/

#To correct access permissions to the database and installation files
sudo chown -Rv ffpu:nogroup /usr/share/ffplayout/db/
sudo chown -Rv ffpu:nogroup /usr/share/ffplayout/
sudo chown -Rv ffpu:nogroup /var/lib/ffplayout/
sudo chown -Rv ffpu:nogroup /etc/ffplayout/
```
